### ISSUE TITLE
User is not informed when Exif is not scrubbed

### ISSUE THREAD

**Issue by opk12** [vector-im/element-android#3539]
1. Send image, uncheck `Send image with the original size`
2. Download, learn that Element auto-strips sensitive EXIF metadata, love it
3. Read no mention of `datetime, make and model, IDs` on the checkbox, infer it is safe to tick (false sense of security)
3. Send image to Mr Evil, do check `Send image with the original size`
4. Sensitive EXIF metadata was sent.

For clarity, lack of communication that EXIF was stripped is safe, the worst case is an inconvenience. This issue is about lack of communication that EXIF was **not** going to be stripped.

**Comment by opk12** [comment]
#465 is about enabling EXIF stripping. Some ideas were described in that issue, such as an explicit option buried deep in the settings that few people will read; automatic choice based on the image source. These do not necessarily address the concern of communicating that EXIF is **not** going to be stripped. An explicit, prominent confirmation dialog or option displayed just before sending would address both issues IMO.

**Comment by est31** [comment]
I think #4248 resolved this, no?

**Comment by opk12** [comment]
@est31 No, the steps from the first comment can be easily verified to produce the same results on the current Element and SchildiChat.

**Comment by opk12** [comment]
@est31 The UI does not make it explicit that scrubbing is intentional. It may well be a bug that goes away with an update.

The label should be amended to

    Send image with the original size (strip metadata)

to allow for a conscious decision about metadata stripping. Some use cases need metadata to be sent.

**Comment by est31** [comment]
So you agree with me that scrubbing happens now, right? Then it's clearly not "User is not informed when Exif is not scrubbed" any more, as exif *is* being scrubbed.

**Comment by basings** [comment]
I am not sure if I should open a separate issue for this because it is somewhat diferent but it is not nice that element strips exif data away. In #465 it was asked to automatically strip exif data away. I do not like that the gps location is lost when sending an image. It would be nice to have the option to preserve all exif data. When I send an image in original size, I actually expect that the original image is send and not that the exif data is removed. When I send a compressed image, I could assume that exif data is removed but even then, I actually assume that only the size is smaller. It is nice having the option to remove exif data to reduce the neccessity of apps like "scrambled exif". It is not nice that I am not able to use element to send an image of a friend I just took to this friend because exif is removed.
